courts_db/utils.py

```diff
@@ -2,7 +2,6 @@
 import os
 import re
 from glob import iglob
-from io import open
```

Comment:
This one was new to me!

https://docs.astral.sh/ruff/rules/open-alias/

Turns out `io.open` is a Python 2 thing